### DISCUSSION TITLE
Instant energy consumption in drive view

### DISCUSSION THREAD

**Discussion by federicotravaini** [CarlosCuezva/dashboards-Grafana-Teslamate#15] [Ideas]
Hello, first of all thank you very much for all these amazing dasborads.

Something that I can't find anywhere is the energy consumption "line" during a drive. I have no idea if this is something that is not collected or what else, but I think it would be nice to have a sort of "energy view" like the one that already is in the car.

Thanks in advance for your time!

**Comment by CarlosCuezva** [comment]
Hello Federico.

Thanks to you and I hope you enjoy the dashboards.

Power consumption is not provided by the Tesla API so Teslamate does not collect any data on it.

Best regards
